pkg/controller/cluster/resource/resource.go

```diff
@@ -151,6 +162,11 @@ func StatefulSetForRack(r scyllav1alpha1.RackSpec, c *scyllav1alpha1.Cluster, si
 									MountPath: naming.SharedDirName,
 									ReadOnly:  false,
 								},
+								{
+									Name:      "scylla-config-volume",
+									MountPath: naming.ScyllaConfigDirName,
+									ReadOnly:  true,
+								},
```

Comment:
Is it necessary to mount it in the sidecar?

Reply:
You mean in the init container?

Reply:
Exactly, miswording on my part.

Reply:
Correct, removing it.